Library/ViewModels/DiscoveryProjectCardViewModel.swift

```diff
@@ -160,3 +173,34 @@ private func projectTags(project: Project, shouldShowPWLTag: Bool, shouldShowCat
 
   return tags
 }
+
+private func facepileData(for friendsList: [User]) -> FacepileViewData? {
+  guard !friendsList.isEmpty else { return nil }
+
+  let avatars = friendsList.prefix(3).map(\.avatar.small).compactMap(URL.init(string:))
+
+  var description: String = ""
+
+  if friendsList.count == 1 {
```

Comment:
A `switch` on the count might read a bit better here. `description` can also be a `let` if the compiler is sure it'll be instantiated.